addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/handler/EvohomeHandler.java

```diff
@@ -49,6 +55,26 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     @Override
     public void initialize() {
```

Comment:
Will do at a later point - I just didn't want to cause too many clashes on our first pull.